jetty-websocket/websocket-jetty-client/src/main/java/org/eclipse/jetty/websocket/client/WebSocketClient.java

```diff
@@ -388,11 +391,48 @@ public synchronized void setStopAtShutdown(boolean stop)
         stopAtShutdown = stop;
     }
 
+    public void setStopTimeout(long stopTimeout)
```

Comment:
Need javadoc: what are the units? Can it be disabled? Can it be set to wait forever?

Reply:
It works exactly the same as `Server` stopTimemout, if it is less or equal to 0 we don't attempt a graceful stop, if it is greater than 0 we use the value.